packages/utils/package.json

```diff
@@ -1,12 +1,13 @@
 {
   "name": "@celo/utils",
-  "version": "0.0.5",
+  "version": "0.0.6-beta",
   "description": "Celo common utils",
   "author": "Celo",
   "license": "Apache-2.0",
   "main": "./lib/index.js",
   "types": "./lib/index.d.ts",
   "scripts": {
+    "prepublishOnly": "yarn build",
```

Comment:
Why `Only` here?

Reply:
From the doc:
> prepublish: Run BEFORE the package is packed and published, as well as on local npm install without any arguments

[source](https://docs.npmjs.com/misc/scripts)

That's actually confusing and `npm` added the `Only` variant, to run **only when publishing**; which is what i wanted.